release.cmd

```diff
@@ -4,6 +4,8 @@ git checkout master
 git pull || echo Could not pull changes from origin && exit /b
 
 dotnet run --project Versionize --framework net6.0 || echo Exiting. Could not create a new version using versionize && exit /b
+git push --follow-tags origin master  || echo Exiting. Could push changes
```

Comment:
Guessing you meant to put _could not_ or _couldn't_.

Reply:
Thanks 👍 the exit was missing as well...